src/test/java/com/zaxxer/nuprocess/DirectWriteTest.java

```diff
@@ -105,7 +107,9 @@ public void onStdout(ByteBuffer buffer, boolean closed)
 
       NuProcessBuilder pb = new NuProcessBuilder(processListener, command);
       NuProcess nuProcess = pb.start();
-      for (int i = 0; i < 1000; i++) {
+      // TODO: given a large i (e.g. 1,000, 10,000), this unit test (testConsecutiveWrites) will
+      //       produce a side-effect on InterruptTest (has problem on Mac OS X, but works on Linux and Win32).
+      for (int i = 0; i < 100; i++) {
```

Comment:
That said, *something* is indeed up with the interrupting processes on OS X.  One out of every few runs of InterruptTest on OS X leaves one, two, or three zombie ``/bin/cat`` processes with *no parent* pid.  These processes are unkillable, even by root with ``kill -9``, the only recourse to get rid of them is reboot.